src/styles/blocks/card.scss

```diff
@@ -0,0 +1,98 @@
+.card {
+  @include smooth;
+
+  border: solid 1px #f3f3f3;
+  width: $card-width;
+  height: $card-height;
+
+  background-image: url($prod-img-source);
+  background-position-y: 32px;
+  background-position-x: 19px;
+  background-size: $prod-img-size;
+  background-repeat: no-repeat;
+
+  &__text {
+    @include smooth;
+
+    line-height: 18px;
+    font-size: $font-size-default;
```

Comment:
Ensure that the `$font-size-default` variable is defined in your SCSS files. If it's not, you need to define it to avoid compilation errors.